packages/pure/src/device/base-device.test.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import BaseDevice from "./base-device"
+import { DeviceType } from "./constants/device-type.enum"
+import BaseMcSerialPortDevice from "./mc-serialport-device"
+import BaseMcUsbDevice from "./mc-usb-device"
+import {
+  McUsbFileType,
+  RequestConfig,
+  ResponseStatus,
+  Endpoint,
+  Method,
+} from "./device.types"
+
+const serialPortDevice = new BaseMcSerialPortDevice(
+  "./path",
+  DeviceType.MuditaPure
+)
+const usbDevice = new BaseMcUsbDevice()
```

Comment:
I think that will be easier to maintain the unit test if you mock dependencies 😉 Imagine that in the next steps the `getFiles` dependencies from the USB source are no mock data like right now.

Reply:
Mocks added in [8045a34](https://github.com/mudita/mudita-center/pull/681/commits/8045a34fa0741889e6bbbacbe34816ca989e3107